integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java

```diff
@@ -7,15 +7,13 @@
 
   import org.json.JSONObject;
   import org.junit.Test;
-  import org.opensearch.sql.legacy.SQLIntegTestCase;
-
+  
```

Comment:
```suggestion
```

Reply:
Addressed in 65df31b37e9b2766d0a5fd4c31745fdc3638d4af